samples/daily-prebuilt/permissions-can-admin/src/pages/api/rooms.js

```diff
@@ -1,28 +1,30 @@
 export default async function handler(req, res) {
+  // This handler only accepts POST requests.
+  const { method } = req;
```

Comment:
There was an eslint error here related to vars being defined inside a `case` statement, which is not actually limited in scope to the `case`. To remove the error we could put it into its own scope block, but then I realized if we just early-out in case of an unsupported method we won't need the `switch` statement at all (and can also reduce nesting this way, which can be a bit easier on reader eyes)